docs/uml/AddDishCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,22 @@
+@startuml
+title Add Dish Sequence Diagram
+
+'actor User
```

Comment:
Just a small remark, for this one, is there a reason why User is commented out? Or maybe you can consider using another participant that is calling this AddDishCommand so that reader knows where is this call initiated from